examples/reshare/src/validator.rs

```diff
@@ -70,7 +70,12 @@ pub async fn run<S>(
         &p2p_namespace,
         SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), config.port),
         SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), config.port),
-        config.bootstrappers.clone().into_iter().collect::<Vec<_>>(),
+        config
+            .bootstrappers
+            .clone()
```

Comment:
nit: this creates an intermediary map that's cloned. Instead of calling `bootstrappers.clone()`, you could do `.map(|(k, v)| (k.clone(), v.clone().into()))` to avoid the intermediary map. Depending on whether `into()` works on references, you might not have to `v.clone()` at all. 

Although since the performance doesn't matter for this one-time function, the current code may be more readable